pan_indent_checker.py

```diff
@@ -70,46 +85,62 @@ def main():
     colorama_init(strip=(not supports_color()))
 
     with open(args.input, 'r+') as file_input:
-        for line_number, line_raw in enumerate(file_input):
+        # Find annotations and reset pointer
+        annotations = find_annotation_blocks(file_input.read())
+        file_input.seek(0)
+
+        # Iterate over lines
+        for line_number, line_raw in enumerate(file_input, 1):
```

Comment:
How has the start value of the enumeration changed without seemingly needing a change to the looping? Was it a bug originally?

Reply:
Oh, yes it was, but wasn't spotted before as the numbering was only used for output.